Makefile

```diff
@@ -2,8 +2,8 @@ ifeq ($(shell uname -m) ,arm64)
 	DOCKER_FLAGS += --platform linux/amd64
 endif
 
-test:
-	@echo "No tests"
+test: build FORCE
+	docker run $(DOCKER_FLAGS) --rm -v=$(CURDIR)/radius:/etc/raddb -v=$(CURDIR)/radius/certs:/etc/raddb/certs -p 1812-1813:1812-1813/udp -p 3000:3000 -p 9812:9812 --name govwifi-frontend-c govwifi-frontend /usr/bin/run-tests.sh
```

Comment:
Why is the second explicit mount of /etc/raddb/certs necessary? a comment below says it is but not why